client/nginx.go

```diff
@@ -141,6 +142,34 @@ type NginxInfo struct {
 	ParentProcessID uint64 `json:"ppid"`
 }
 
+// Caches is a map of cache stats by cache zone
+type Caches = map[string]HTTPCache
+
+// HTTPCache represents a zone's HTTP Cache
+type HTTPCache struct {
+	Size        uint64
+	MaxSize     uint64 `json:"max_size"`
+	Cold        bool
+	Hit         CacheResponses
+	Stale       CacheResponses
+	Updating    CacheResponses
+	Revalidated CacheResponses
+	Miss        CacheResponses
+	Expired     CacheResponsesWritten
+	Bypass      CacheResponsesWritten
+}
+
+type CacheResponses struct {
```

Comment:
could you add doc strings for the two new structs, since they are exported?

I think it is a bit of a challenge to describe them properly, because their meaning depend on the metric they are about. Considering that, I can suggest the following generic descriptions:

```
// CacheResponses are basic cache stats.
type CacheResponses struct

// CacheResponsesWritten are extended cache stats.
type CacheResponsesWritten struct
```

Although, I think it would make sense to rename the types as well:
```
// CacheStats are basic cache stats.
type CacheStats struct

// ExtendedCacheStats are extended cache stats.
type ExtendedCacheStats struct
```